src/plugins/data/common/search/aggs/utils/date_interval_utils/parse_interval.ts

```diff
@@ -7,21 +7,32 @@
  */
 
 import { find } from 'lodash';
-import moment, { unitOfTime } from 'moment';
-import dateMath from '@elastic/datemath';
+import moment from 'moment';
+import dateMath, { Unit } from '@elastic/datemath';
```

Comment:
no functional changes in this file right except for the code cleanup ?

Reply:
yep, it was wrong type